Blueprints/BlueprintDefinitions/vs2022/AnnotationsFramework/template/src/BlueprintBaseName.1/CalculatorService/ICalculatorService.cs

```diff
@@ -0,0 +1,40 @@
+namespace BlueprintBaseName._1
```

Comment:
nit: could we stick to using top-level statements?

Reply:
Done